resources/carrier/en/98.txt

```diff
@@ -21,10 +21,18 @@
 98931|MTCE
 98932|Taliya
 98933|Irancell
-98934|TKC
+98934|TeleKish
 98935|Irancell
 98936|Irancell
 98937|Irancell
 98938|Irancell
 98939|Irancell
 98990|IR-MCI
+98991|IR-MCI
+989944|Ansarestan
```

Comment:
Isn't it Anarestan? http://www.anarestan.com/en/about/
https://fa.wikipedia.org/wiki/%D8%B4%D9%85%D8%A7%D8%B1%D9%87%E2%80%8C%D9%87%D8%A7%DB%8C_%D8%AA%D9%84%D9%81%D9%86_%D8%AF%D8%B1_%D8%A7%DB%8C%D8%B1%D8%A7%D9%86#.D9.84.D9.88.D8.AA.D9.88.D8.B3.E2.80.8C.D8.AA.D9.84_LOTUSTEL

Please change if you agree on this.

Reply:
The site that you mentoined, is anarestan's official website. (www.anarstan.com/en/about)
But Anarestan isn't only an MVNO. It's a company that working on mobile and children, teens and students(Wide from Sepecial sim cards for children via Anarestan MVNO to Building an Android Market that is for Children and Teacher and Students) to special mobile internet for children(Bad contents for children blocked). In addition, anarestan offering mobile apps that are for mobile based elearning for students and schools.
All of these products are working under anarestan.
Note: That link you mentoined from wikipedia, is for LOTUSTEL(+9899900) not anarestan.
You can visit this page about anarestan sim cards(Anarestan MVNO):
http://www.anarestan.com/en/service/anarestans-sim-card/

Anarestan simcards in first steps was prepaid IR-MCI simcards with special services via anarestan(For students) with prefix numbers of IR-MCI. But when iranian regulatory announced for MVNOs, anarestan also requsted to founding it's own MVNO and currently delivering anarestan simcards with it's own MVNO brand and prefix number(+989944) not IR-MCI.
For more services and products from anarestan, you can check it's website's services menu.

Reply:
Thanks for providing the details. 
As you confirmed it is Anarestan, not An**s**arestan (which is mentioned in the file). Can you please update this so that I can provide the final approval?